force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -162,17 +205,16 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static String repeatString(String inputString, Integer repeatCount) {
         // Initialize the result String
+        String resultString='';
 
         // Use a for loop to append the inputString to the result string repeatCount times
-        /*
-        for () {
-
-            // If it is not the last iteration, add a semicolon to separate the strings
+        for (integer i=0;i<repeatCount;i++) {
+            resultString+=inputString;
+            if(i!=repeatcount-1){
+                resultString+='; '; // If it is not the last iteration, add a semicolon to separate the strings
```

Comment:
I like that this is a separate section to only add the separator, and not divided into two separate parts of logic, both adding inputString.